release.py

```diff
@@ -0,0 +1,848 @@
+    pr = repo.create_pull(
+        title=f'chore: update changelog and versions for {tag} release',
+        body=f'This PR prepares the release of version {tag}.',
+        head=f'{gh_client.get_user().login}:{new_branch}',  # "your_username:new_branch"
+        base=branch,
+    )
+    logger.info(f'Created PR: {pr.html_url}')
+    countdown('Opening the PR in: ', 3)
+    webbrowser.open(pr.html_url)
+
+
+def publish_draft_release(owner: str, repo_name: str):
+    """Publish the draft release."""
+    org = gh_client.get_organization(owner)
+    repo = org.get_repo(repo_name)
+
+    releases = repo.get_releases()
+    draft = None
+    for release in releases:
+        if release.draft:
+            draft = release
+            break
+
+    if not draft:
+        logger.error('No draft release found. Please create a draft release first.')
+        return
+
+    while True:
+        confirm = (
+            input(f"Confirm publishing draft release '{draft.html_url}'? [y/N]: ").strip().lower()
+        )
+
+        if confirm.lower() == 'n':
+            logger.warning('Release publish canceled.')
+            exit(0)
+
+        if confirm == 'y':
+            break
+
+    logger.info(f'Publishing draft release: {draft.title} {draft.html_url}')
+    draft.update_release(name=draft.title, message=draft.body, draft=False)
+
+
+def post_release(owner: str, repo_name: str, branch: str):
+    """Post-release actions: update version files and create a PR."""
+    new_branch = 'post-release'
+    local_branch = subprocess.run(
+        ['/usr/bin/git', 'branch', '--list', new_branch],
+        capture_output=True,
+        text=True,
+        check=True,
+    ).stdout.strip()
+    remote_branch = subprocess.run(
+        ['/usr/bin/git', 'ls-remote', '--heads', 'origin', new_branch],
+        capture_output=True,
+        text=True,
+        check=True,
+    ).stdout.strip()
+    exist = local_branch or remote_branch
+    if exist:
+        logger.error(
+            f'Branch "{new_branch}" already exists in the current repository. '
+            'Please double check and delete it first before post release'
+        )
+        exit(1)
+
+    subprocess.run(['/usr/bin/git', 'fetch', 'upstream'], check=True)
+    subprocess.run(['/usr/bin/git', 'checkout', branch], check=True)
+    subprocess.run(['/usr/bin/git', 'merge', f'upstream/{branch}'], check=True)
+
+    org = gh_client.get_organization(owner)
+    repo = org.get_repo(repo_name)
+
+    update_versions_for_post_release(repo, branch)
+
+    subprocess.run(['/usr/bin/git', 'checkout', '-b', new_branch], check=True)
+    for file in VERSION_FILES.values():
+        subprocess.run(['/usr/bin/git', 'add', file], check=True)
+    subprocess.run(
+        ['/usr/bin/git', 'commit', '-m', 'chore: update versions after release'], check=True
+    )
+    subprocess.run(['/usr/bin/git', 'push', 'origin', new_branch], check=True)
+    pr = repo.create_pull(
+        title='Post Release',
```

Comment:
This will have to be something like `chore: adjust versions after release` so that it fits with the conventional commits.